invenio_rdm_records/oaiserver/services/services.py

```diff
@@ -97,10 +98,10 @@ def create(self, identity, data, uow=None):
             context={"identity": identity},
             raise_errors=True,
         )
-
         self._validate_spec(valid_data["spec"])
+        system_created = valid_data["spec"].startswith(self.reserved_prefix)
 
-        new_set = OAISet(**valid_data)
+        new_set = OAISet(**valid_data, system_created=system_created)
```

Comment:
the `_get_one` method should use the service instead?

Reply:
I am sorry but I am not sure I understand this, could you explain it a bit more? thanks!